R/manual_layer_node.R

```diff
@@ -357,58 +357,66 @@ Node <- R6::R6Class(
 ##'
 ##' @family {manual-layer functions}
 ##' @export
-compute <- function(delayed_function, namespace=NULL, timeout_seconds=NULL, verbose=FALSE, force_all_local=FALSE) 0
-setMethod("compute", signature(delayed_function = "Node"), function(delayed_function, namespace=NULL, timeout_seconds=NULL,
+setGeneric("compute",
+           function(node, namespace=NULL, timeout_seconds=NULL, verbose=FALSE, force_all_local=FALSE)
```

Comment:
Reason for change from `delayed_function` to `node`:

* In this package, `Node` and `DAG` are implementation details; the [user-facing surface](https://tiledb-inc.github.io/TileDB-Cloud-R/articles/TaskGraphs.html) centers on factory methods with `delayed` in the name. So, I thought to call this `delayed_function`.
* However, it's not really a function, it's a `Node` object and perhaps calling it `delayed_function` increased, rather than reduced, clarity ...

Reply:
Maybe `delayed_compute` to express both components?

Reply:
Or maybe `delayed_*` is only for args?  Your call....